stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpConstants.java

```diff
@@ -228,6 +228,11 @@
     public static final int PROXY_USER_NAME_INDEX = 1;
     public static final int PROXY_PASSWORD_INDEX = 2;
 
+    //Connection Throttling struct indexes
+    public static final int CONNECTION_THROTTLING_STRUCT_INDEX = 4;
+    public static final int MAX_ACTIVE_CONNECTIONS_INDEX = 0;
```

Comment:
Shall we rename variable as "CONNECTION_THROTTLING_MAX_ACTIVE_CONNECTIONS_INDEX"  or  "THROTTLING_MAX_ACTIVE_CONNECTIONS_INDEX" to relate the struct?